### DISCUSSION TITLE
Lesson 6 : Foundry installation step for window user was not given?

### DISCUSSION THREAD

**Discussion by LeeJianXing** [Cyfrin/foundry-full-course-cu#928] [Q&A] [answered]
At 6:46:15 of the course, Patrick said the window installation guide will be given later but i could not found it in the video so i want to know that is it not given or i just missed it in the video

**Comment by 0xusmanf** [comment, marked as answer]
No, you did not miss it, some parts of the tutorial are out of sequence so maybe Patrick was mentioning about the part that Vasiliy had already explained at [06:34:32](https://youtu.be/umepbfKp5rI?t=23672), as after installing and running WSL the installation process is same for windows.

Just follow along with Patrick to install Foundry.

**Reply by LeeJianXing** [reply]
Thank you.I was wondering about where is the window version "WSL" of Foundry setup guide is after patrick mention.Looks like it is out of sequence,since i was already have WSL installed i followed along the foundry setup and all good
